### PR TITLE
New package: spt-0.4

### PR DESCRIPTION
**Pull request by ibabushkin**
A simple pomodoro timer. Also quite useful at times. `libnotify` integration compiled in.

### CODE REVIEW

File: srcpkgs/spt/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'spt'
+pkgname=spt
+version=0.4
+revision=1
+short_desc="Simple pomodoro timer"
+maintainer="Inokentiy Babushkin <inokentiy.babushkin@googlemail.com>"
+license="Custom"
```
Comment (Duncaen) [inline srcpkgs/spt/template]: ISC afaik

File: srcpkgs/spt/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="pkg-config"
+makedepends="libnotify-devel glib-devel"
+depends="libnotify"
```
Comment (Duncaen) [inline srcpkgs/spt/template]: This is not necessary if it links against `libnotify.so`, xbps-src detects it. 
See https://travis-ci.org/voidlinux/void-packages/jobs/207541866#L649.

File: srcpkgs/spt/template (hunk 1 above)
Comment (ibabushkin) [inline srcpkgs/spt/template]: Thanks!

File: srcpkgs/spt/template (hunk 2 above)
Comment (ibabushkin) [inline srcpkgs/spt/template]: Yes, you're right. Fixed now.

**Comment by ibabushkin** [comment]
Is there any further change needed on this one? Or should we wait on #5917 to land?

**Comment by Vaelatern** [comment]
The install ought to be taking care of the vman itself, can you confirm?

**Comment by ibabushkin** [comment]
Yes, it seems to. I'll remove it and update the commit.

**Pull request merged**